bridge/client/app/_views/ktb-settings-view/ktb-integration-view/ktb-modify-uniform-subscription/ktb-modify-uniform-subscription.component.ts

```diff
@@ -208,6 +215,7 @@ export class KtbModifyUniformSubscriptionComponent implements OnDestroy {
         };
       }),
       tap((data) => {
+        this.webhookFormTouched = true;
```

Comment:
This is at the wrong place. webhookFormTouched is set to true after the data is fetched not when the data is changed